src/features/misc/helpers/fetchTokenMetadata.ts

```diff
@@ -10,10 +10,6 @@ import { TokenStandard } from '$/features/tokenGatedRooms/types'
 import { MiscAction } from '$/features/misc'
 
 async function getURIs(tokenIds: string[], contract: Contract) {
-    if (tokenIds.length === 0) {
```

Comment:
This is controversial because I changed boom into no boom.

Technically if we fetch metadata there's no reason to prevent fetching no token ids. `uris` is going to be empty. That's it.

I believe non-empty token id list is more of a problem for the policy deployment.